API/contact/Contact.php

```diff
@@ -0,0 +1,91 @@
+<?php
+    require_once __DIR__ . '/../utils/JsonUtils.php';
+
+    class Contact
+        extends JsonDeserializer
+        implements JsonSerializable  
+    {
+        public int $ID;
```

Comment:
I would want to use contactID instead of ID but currently our database has ID... So, the way JsonDeserializer works, it basically looks at "Exact" match of the "column name" from database to the "field name" inside of the class... 

In other words, if database would want to change their column names (ID to ContactID/UserID) then I can change the class field names

Reply:
![image](https://user-images.githubusercontent.com/38502074/190281485-b9805f38-162d-4891-8a14-0bde576e4547.png)

Reply:
So, in order for property_exists() to work, field/property name should be the same as "key" name... and key name is the name of the column in database.